### ISSUE TITLE
Why Accuracy and Loss are not corresponding?

### ISSUE THREAD

**Issue by dlrPHS2** [shekkizh/FCN.tensorflow#50]
I have implemented the code and extended it according to the statistics (Accuracy and Loss). However, I expect a correspondence between Accuracy and Loss in TensorBoard. Unfortunately, the graphs do not agree (See Attachment). Does anyone know why?
Thanks in Advance!
![graph](https://user-images.githubusercontent.com/27953501/27804370-e1fd0d28-602e-11e7-8536-e350c3dcd83b.png)

**Comment by shekkizh** [comment]
Try reading on loss functions used to train machine learning algorithms. You want a smooth loss function for training with backprop.

**Comment by varungupta31** [comment]
Hey @dlrPHS2 did you train the model using `debug=True`?
I'm trying to visualize training metrics like accuracy, mean accuracy, mean iou, f.w iou but am only getting an 'entropy' curve on tensorboard? Were you able to log/visualize these metrics, how?

![image](https://user-images.githubusercontent.com/51288316/159124615-452f07cc-fd74-4ff2-bdb9-c3850f24879f.png)


Thank you.
